ui/src/Panels/Producer/Producer.view.js

```diff
@@ -0,0 +1,130 @@
+/* eslint-disable react/jsx-no-bind */
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import { debounce, get } from 'lodash-es';
+
+import { translations } from './Producer.assets.js';
+import { CONSTANTS, idAttributeGenerator } from 'Utils';
+import { Counter, ProducerMessages } from 'Groups';
+import { ProducerMessage } from 'Elements';
+import {
+  useTranslate,
+  useToggle,
+  useKafkaVertxWebSocket,
+} from 'ReactCustomHooks';
+import { Button, TextInput } from 'carbon-components-react';
+
+const Producer = (props) => {
+  const {
+    getProducerWebsocket,
+    topic,
+    maxNumberOfMessages,
+    className,
+    ...others
+  } = props;
+  const classesToApply = clsx('Producer', { [className]: className });
+
+  const translate = useTranslate(translations);
+  const [producerRunning, toggleProducerRunning] = useToggle(false);
+  const {
+    start,
+    stop,
+    isReady,
+    isRunning,
+    messages,
+    totalSuccessMessages,
+  } = useKafkaVertxWebSocket(getProducerWebsocket, maxNumberOfMessages);
+  const [messageValue, setMessageValue] = useState(
+    translate('MESSAGE_VALUE_DEFAULT', {}, true)
+  );
+  const debouncedSetMessageValue = debounce(setMessageValue, 100);
+
+  const onButtonClick = () => {
+    if (producerRunning) {
+      stop();
+    } else {
+      start({ value: messageValue });
```

Comment:
the key `value` needs to be `custom` as that is what the backend expects

Reply:
Thanks! Was convinced that wasn't correct but couldn't spot what it was supposed to be.